src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,682 @@
+				eventArgument="ui.item" argType="object" Gets a reference to the list item which is selected.
+			*/
+			dropDownItemSelected: "dropDownItemSelected"
+			/* cancel="false" Fired after the editor's text has been changed. It can be raised when keyUp event occurs,
+				when the clear button is clicked or when an item from a list is selected.
+				```
+				//Delegate
+				$(".selector").on("%%WidgetNameLowered%%textchanged", function (evt, ui) {
+					...
+				});
+
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					textChanged: function (evt, ui) {
+					...
+					}
+				});
+				```
+				eventArgument="evt" argType="event" jQuery event object.
+				eventArgument="ui.owner" argType="object" Gets a reference to the editor.
+				eventArgument="ui.text" argType="string" Gets a reference to the new text.
+				eventArgument="ui.oldText" argType="string" Gets a reference to the old text.
+			*/
+		},
+		_create: function () { // igTimePicker
+			$.ui.igDateEditor.prototype._create.call(this);
+		},
+		_initialize: function () { // igTimePicker
+			if (this.options.timeDisplayFormat) {
+				this.options.timeDisplayFormat = this._parseTimeMask(this.options.timeDisplayFormat);
+				this.options.dateDisplayFormat = this.options.timeDisplayFormat;
+			} else {
+				this.options.dateDisplayFormat = "time";
+			}
+
+			if (this.options.timeInputFormat) {
+				this.options.timeInputFormat = this._parseTimeMask(this.options.timeInputFormat);
+				this.options.dateInputFormat = this.options.timeInputFormat;
+			} else {
+				this.options.dateInputFormat = "time";
+			}
+
+			this._super();
+
+			if (this.options.minTimeValue) {
+				this.options.minValue = this._parseDateFromMaskedValue(this.options.minTimeValue);
+			}
+
+			if (this.options.maxTimeValue) {
+				this.options.maxValue = this._parseDateFromMaskedValue(this.options.maxTimeValue);
+			}
+
+			var buttons = this.options.buttonType.toString().split(/[\s,]+/);
+
+			if ($.inArray("dropdown", buttons) === 0) {
+				this._populateDropDown(this.options.minTimeValue, this.options.maxTimeValue);
+			}
+		},
+		getSelectedListItem: function () {
+			/* Gets the selected list item.
+			```
+			var selectedItem =  $(".selector").%%WidgetName%%("getSelectedListItem");
+			```
+				returnType="$" Selected list item.*/
+			return $.ui.igTextEditor.prototype.getSelectedListItem.call(this);
+		},
+		dropDownVisible: function () {
+			/* Returns the visibility state of the calendar.
+			```
+			$(".selector").igTimePicker("dropDownVisible");
+			```
+				returnType="bool" The visibility state of the calendar. */
+			return $.ui.igTextEditor.prototype.dropDownVisible.call(this);
+		},
+		dropDownButton: function () {
+			/* Returns a reference to the drop-down button UI element of the editor.
+			```
+			var button = $(".selector").%%WidgetName%%("dropDownButton");
+			```
+				returnType="$" Returns reference to jquery object. */
+			return $.ui.igTextEditor.prototype.dropDownButton.call(this);

```

Comment:
Use `_super` instead. Btw, do you really need the overrides, since they do nothing different?